helms/odahu-flow-opa/policies/permissions.rego

```diff
@@ -8,14 +8,22 @@ permissions := {
     	[".*", "api/v1/model/packaging.*"],
     	[".*", "api/v1/model/training.*"],
     	["GET", "api/v1/configuration.*"],
-    	["GET", "api/v1/connection.*"],
+    	["GET", "api/v1/connection/*[a-zA-Z0-9-]*/*$"],
     	["GET", "api/v1/packaging/integration.*"],
-    	["GET", "api/v1/toolchain/integration.*"]
+    	["GET", "api/v1/toolchain/integration.*"],
+    	["GET", "health"]
     ],
   roles.admin : [
       [".*", ".*"]
   ],
   roles.viewer : [
-      ["GET", ".*"]
+    	["GET", "api/v1/model/deployment.*"],
+    	["GET", "api/v1/model/packaging.*"],
+    	["GET", "api/v1/model/training.*"],
+    	["GET", "api/v1/configuration.*"],
+    	["GET", "api/v1/connection/*[a-zA-Z0-9-]*/*$"],
+    	["GET", "api/v1/packaging/integration.*"],
+    	["GET", "api/v1/toolchain/integration.*"],
+    	["GET", "health"]
```

Comment:
Can you check the following URL returns 403 for a user with viewer role: `/api/v1/connection/health/decrypted`?
If it doesn't, then maybe we should mark the start and of a line explicitly in every URL regex?

Reply:
yes, returns.